client/client.go

```diff
@@ -1719,3 +1731,45 @@ func trimHTTPPrefix(str string) string {
 	str = strings.TrimPrefix(str, "https://")
 	return str
 }
+
+func (c *client) LoadGlobalConfig(ctx context.Context, names []string) ([]string, error) {
+	r, err := c.getClient().LoadGlobalConfig(ctx, &pdpb.LoadGlobalConfigRequest{Names: names})
+	if err != nil {
+		return nil, err
+	}
+	if r.GetOk() {
+		return r.GetValues(), nil
+	} else {
+		return nil, errors.New(fmt.Sprintf("result of load global config is not ok, maybe any of the key is not found: %v", names))
+	}
+}
+
+func (c *client) StoreGlobalConfig(ctx context.Context, name string, value string) error {
+	_, err := c.getClient().StoreGlobalConfig(ctx, &pdpb.StoreGlobalConfigRequest{Name: name, Value: value})
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func (c *client) WatchGlobalConfig(ctx context.Context) (chan GlobalConfigItem, error) {
+	receiver := make(chan GlobalConfigItem, 3)
+	r, err := c.getClient().WatchGlobalConfig(ctx, &pdpb.WatchGlobalConfigRequest{})
+	if err != nil {
+		return receiver, err
+	}
+	go func() {
```

Comment:
```suggestion
	go func() {
	    defer close(receiver)
```

should close the channel, otherwise, the user will not know this watching is download.